src/main/java/com/leverx/learningmanagementsystem/web/oauth/token/service/TokenService.java

```diff
@@ -6,7 +6,6 @@
 import org.springframework.cache.annotation.CachePut;
 import org.springframework.cache.annotation.Cacheable;
 import org.springframework.context.annotation.Profile;
```

Comment:
I see that in this class (which is quite generic), `DestinationTokenRequest` is used as parameter. It should be replaced with more generic logic